docs/src/main/mdoc/certificates.md

```diff
@@ -0,0 +1,34 @@
+---
+id: security
+title: Security & Certificates
+---
+
+## Security: certificates, trust stores, and passwords 
+
+The `KafkaCredentialStore` can be used to create the necessary.
+
+The parameters passed in are string representations of the client private key, client certificate
+and service certificate. the `properties` field in `KafkaCredentialStore` can then be applied to
+any of the `*Settings` classes by using the `withProperties(kafkaCredentialStore.properties)`.
```

Comment:
How about adding `withCredentials(store: KafkaCredentialsStore)` to the `*Settings` classes?

Reply:
Oooh nice, good call @bplommer. Makes it a bit clearer and nicer than passing around property maps.